modules/cnv-viewing-node-network-state.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-node-network-state.adoc
+
+[id="cnv-viewing-node-network-state_{context}"]
+= Viewing node network state
```

Comment:
I would prefer "observing" since I can't pronounce this :( And maybe "network state of a node"?

Reply:
I have updated the assembly to 'Observing' but openshift docs uses convention of 'viewing' for modules. 
Updated to 'Viewing the network state of a node'